core/src/main/java/org/bitcoinj/wallet/Wallet.java

```diff
@@ -2613,9 +2613,17 @@ private void maybeMovePool(Transaction tx, String context) {
     }
 
     /**
-     * Calls {@link Wallet#commitTx} if tx is not already in the pending pool
+     * Updates the wallet with the given transaction: puts it into the pending pool, sets the spent flags and runs
+     * the onCoinsSent/onCoinsReceived event listener.
+     * <p>
+     * Triggers an auto save.
```

Comment:
Note: auto-saving needs to be enabled.

Reply:
I added an "(if enabled)" to make this more clear.